bioimageio/core/_resource_tests.py

```diff
@@ -35,13 +35,18 @@ def test_model(
     source: Union[v0_5.ModelDescr, PermissiveFileSource],
     weight_format: Optional[WeightsFormat] = None,
     devices: Optional[List[str]] = None,
-    decimal: int = 4,
+    absolute_tolerance: float = 1e-4,
```

Comment:
```suggestion
    absolute_tolerance: float = 1.5e-4,
```
to not be stricter than before...